tensorboard/components/tf_data_selector/tf-data-selector.html

```diff
@@ -40,14 +40,16 @@
       if="[[!_canCompareExperiments(_comparingExps)]]"
       restamp
     >
-      <tf-data-select-row no-experiment persistence-number="0">
-      </tf-data-select-row>
+      <tf-data-select-row
+        no-experiment
+        persistence-id="default"
```

Comment:
But what will happen when we reach experiment ID 29170397297?

I'd probably use something with a URL-safe character outside [0-9a-z], like an underscore.

Reply:
Done! Thanks for catching that